pandas/tseries/tests/test_timeseries.py

```diff
@@ -3079,6 +3079,42 @@ def test_datetime64_with_DateOffset(self):
             assert_func(result, exp)
             assert_func(result2, exp)
 
+            s = klass([Timestamp('2000-01-15 00:15:00', tz='US/Central'),
```

Comment:
can you move this function into a separate test class (its getting too long), then you can also have simpler names for the class. I think further move this to ``test_offsets.py``.